### PR TITLE
[Mosip-30417] [Mosip_30241] adding properties

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: registration-processor-default.properties [hunk 1]
```diff
@@ -989,6 +990,8 @@ landing.zone.account.name=landing-zone
 landing.zone.type=DMZServer
```
Comment (sowmya695) [inline registration-processor-default.properties]: Please update these properties in this line only,dont add new

**Review by sowmya695 — CHANGES_REQUESTED** [review]
please fix it

File: registration-processor-default.properties (hunk 1 above)
Comment (Khuddusshariff0022) [reply registration-processor-default.properties]: Done

File: registration-processor-default.properties [hunk 2]
```diff
@@ -985,10 +986,13 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
...
 auth.server.admin.allowed.audience=mosip-regproc-client,mosip-admin-client,mosip-resident-client,mosip-reg-client
 mosip.regproc.cbeff-validation.mandatory.modalities=Right,Left,Left RingFinger,Left LittleFinger,Right RingFinger,Left Thumb,Left IndexFinger,Right IndexFinger,Right LittleFinger,Right MiddleFinger,Left MiddleFinger,Right Thumb,EXCEPTION_PHOTO
-landing.zone.account.name=landing-zone
-landing.zone.type=DMZServer
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
 mosip.regproc.landing.zone.fixed.delay.millisecs=86400000
 mosip.regproc.landing.zone.inital.delay.millisecs=300000
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
```
Comment (sowmya695) [inline registration-processor-default.properties]: Please remove the new properties in line 993 and 994

Comment (Khuddusshariff0022) [reply registration-processor-default.properties]: dene

**Review by sowmya695 — APPROVED** [review]

**Review by Rakshitha650 — APPROVED** [review]

File: registration-processor-default.properties [hunk 3]
```diff
@@ -642,6 +642,7 @@ mosip.regproc.demo.dedupe.message.expiry-time-limit=${mosip.regproc.common.stage
 mosip.regproc.demo.dedupe.server.port=8091
 mosip.regproc.demo.dedupe.eventbus.port=5717
 mosip.regproc.demo.dedupe.server.servlet.path=/registrationprocessor/v1/demodedupe
+mosip.regproc.demo.dedupe.trim-whitespaces.simpleType-value:false
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: = to be used

File: registration-processor-default.properties [hunk 4]
```diff
@@ -985,11 +986,12 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
 mosip.role.registration.getPostsyncv2=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER,REGISTRATION_SUPERVISOR,RESIDENT
 auth.server.admin.allowed.audience=mosip-regproc-client,mosip-admin-client,mosip-resident-client,mosip-reg-client
 mosip.regproc.cbeff-validation.mandatory.modalities=Right,Left,Left RingFinger,Left LittleFinger,Right RingFinger,Left Thumb,Left IndexFinger,Right IndexFinger,Right LittleFinger,Right MiddleFinger,Left MiddleFinger,Right Thumb,EXCEPTION_PHOTO
-landing.zone.account.name=landing-zone
-landing.zone.type=DMZServer
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
```
Comment (vishwa-vyom) [inline registration-processor-default.properties:991]: can we add some comments here about the rename here, which will help people when they are upgrading from 1.1.5.5 to 1.2.0.1.

File: registration-processor-default.properties [hunk 5]
```diff
@@ -985,11 +986,12 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
...
 mosip.regproc.landing.zone.fixed.delay.millisecs=86400000
 mosip.regproc.landing.zone.inital.delay.millisecs=300000
 
+
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: Lets remove this

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 5 above)
Comment (Khuddusshariff0022) [reply registration-processor-default.properties]: space removed

File: registration-processor-default.properties (hunk 4 above)
Comment (Khuddusshariff0022) [reply registration-processor-default.properties:991]: comment added.

File: registration-processor-default.properties (hunk 3 above)
Comment (Khuddusshariff0022) [reply registration-processor-default.properties]: done

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**